exchanges/graphcache/src/ast/schemaPredicates.test.ts

```diff
@@ -1,6 +1,10 @@
 import { buildClientSchema } from 'graphql';
 import * as SchemaPredicates from './schemaPredicates';
 
+let warnSpy: jest.SpyInstance;
+beforeEach(() => (warnSpy = jest.spyOn(console, 'warn')));
```

Comment:
Small note. Warnings were an afterthought in Graphcache unfortunately 😅 so in our tests we started testing for them similarly to this and prevented the console from being flooded.
So we already have a global spy: https://github.com/FormidableLabs/urql/blob/5295e33e6d17016912e12f70acd79ac4a51b3a95/scripts/jest/setup.js#L18

So it’s possible now to just run assertions against `console.warn` and `console.warn.mock` directly

Reply:
Done - thanks for the pointer!